codechain/config/mod.rs

```diff
@@ -439,6 +435,11 @@ impl Mining {
         if let Some(work_queue_size) = matches.value_of("work-queue-size") {
             self.work_queue_size = Some(work_queue_size.parse().map_err(|_| "Invalid size")?);
         }
+        if self.author.is_some() || self.engine_signer.is_some() {
```

Comment:
`disable` field of `config::Mining` is weird because we cannot set `disable` via a config file.
Let's remove this field and make add a method to check it.

Reply:
Ok, that seems great